### PR TITLE
phased array example

### PR DESCRIPTION
**Pull request by talg2324**

### CODE REVIEW

File: examples/us_bmode_phased_array/us_bmode_phased_array.ipynb
[the remark this answers is not in the document]
Comment (waltsims) [reply examples/us_bmode_phased_array/us_bmode_phased_array.ipynb:1]: Hey David, 

I have performed the following changes:

- Added install command in the notebook
- Set RUN_SIMULATION to default True
- Created a Python script copy of the notebook with nbconvert

Concerning your other comments, I think @talg2324's implementation adheres quite well to what was performed in the MATLAB example (Screenshot below). What do you think? 
image

**Review by waltsims — COMMENTED** [review]
thanks for the PR

File: examples/us_bmode_phased_array/us_bmode_phased_array.ipynb
[the remark this answers is not in the document]
Comment (talg2324) [reply examples/us_bmode_phased_array/us_bmode_phased_array.ipynb:1]: Thanks @waltsims, looks like you got back to this before me.

@djps re
> * would be good to have a python script of this as well as the notebook.

I saw you guys had a few of these in other notebooks but didn't see it on the requirements for example README. Maybe add it? I think its a nice touch, just should be clear.

I tried to mimic the other examples as much as possible but I wonder why you prefer to upload non-evaluated notebooks? This is something I find quite annoying about the original kwave source-code, I find myself going back and forth between the website and the matlab scripts when starting a new project.

[the remark this answers is not in the document]
Comment (waltsims) [reply examples/us_bmode_phased_array/us_bmode_phased_array.ipynb:1]: Hey @talg2324 talg2324,

Good point, I've updated the instructions to include the direction to add a mirrored script in the directory as well.

The reason we clear the notebook output is to keep binary data out of our git history.  I think its relatively straight forward to quickly run the examples in Colab no?

[the remark this answers is not in the document]
Comment (talg2324) [reply examples/us_bmode_phased_array/us_bmode_phased_array.ipynb:1]: Depending on your compute, some of the examples can take a while to run. This notebook for example would call kspaceFirstOrder3D 32 times before you could have see an image

I do understand it keeps the repo smaller and I agree colab is a nice alternative.

Thanks for clarifying the instructions

**Comment by waltsims** [comment]
- [ ] check symmetry of visualization @djps

**Comment by waltsims** [comment]
Thanks for the PR!

**Review by waltsims — APPROVED** [review]
looks good now.

**Pull request merged**